src/components/dropdown/dropdown.stories.svelte

```diff
@@ -52,7 +52,9 @@
 </Template>
 
 <Story name="Slots" let:args>
-  <SlotInfo description="The dropdown provides several slots for customization">
+  <SlotInfo
+    description="The dropdown provides several slots for customization. It accepts 'leo-menu-item' and 'leo-option.' 'Leo-menu-item' is mainly used for context menus, which are contained within a menu, while 'leo-option' is used for selectable choices."
```

Comment:
nit: a "context menu" is the whole menu. I would suggest that dropdown menu should only support leo-option, and in ButtonMenu story we can suggest both